src/Workspaces/Core/Portable/CodeGeneration/Symbols/CodeGenerationParameterSymbol.cs

```diff
@@ -68,6 +68,10 @@ public override TResult Accept<TResult>(SymbolVisitor<TResult> visitor)
 
         public ImmutableArray<CustomModifier> CustomModifiers => ImmutableArray.Create<CustomModifier>();
 
+        public bool IsRefScoped => false;
+
+        public bool IsValueScoped => false;
```

Comment:
interesting name.  curious what the intuition is on 'value scoped'

Reply:
The name is intended to indicate the members of the `ref struct`, that is, the value of the `ref struct`, does not escape the current method.